src/inverseRobot.js

```diff
@@ -25,6 +25,15 @@
  */
 function inverseRobot(robot) {
   // write code here
+  const newRobot = {};
+
+  for (const key in robot) {
+    if (newRobot.hasOwnProperty(robot[key])) {
+      return null;
+    } newRobot[robot[key]] = key;
```

Comment:
```suggestion
    if (newRobot.hasOwnProperty(robot[key])) {
      return null;
    } 
    newRobot[robot[key]] = key;
```
move to the new line